FBSDKCoreKit/FBSDKCoreKit/Internal/BridgeAPI/FBSDKBridgeAPI.m

```diff
@@ -123,15 +127,28 @@ - (BOOL)application:(UIApplication *)application
       if (_authenticationSession != nil) {
         [_authenticationSession cancel];
         _authenticationSession = nil;
+
+        // This check is needed in case another sdk / message / ad etc... tries to open the app
+        // during the login flow.
+        // This dismisses the authentication browser without triggering any login callbacks.
+        // Hence we need to explicitly call the authentication session's completion handler.
+        if (!canOpenURL) {
+          NSString *errorMessage = [[NSString alloc]
+                                    initWithFormat:@"Login attempt cancelled by alternate call to openURL from: %@",
+                                    url];
+          NSError *loginError = [[NSError alloc]
+                                 initWithDomain:@"com.facebook.sdk.login"
```

Comment:
Shouldn't this use the `FBSDKLoginErrorDomain` constant?

Reply:
I avoided adding that import because I didn't want to add another link between Core and Login-Kit but at this point it's probably better to have that than the duplication. Also these classes are pretty coupled so I'm not sure it'll make it that much harder to eventually uncouple... do you think it makes sense to add a LoginConstants section to the CoreKit constants and duplicate them there? Alternately just move them into CoreKit since LoginKit relies on core anyway?